plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.test.tsx

```diff
@@ -67,14 +94,68 @@ const sampleEntities: Entity[] = [
   },
 ];
 
+const getEntitiesByRefs = jest.fn(async ({ entityRefs }) => ({
+  items: entityRefs.map((e: string) =>
+    ownerEntities.find(f => f.metadata.name === e),
```

Comment:
Name isn't the same as ref though, i know this is just a test but still, i'd expect the `ownerEntities` to have a namespace given as well, and then to check if `stringifyEntityRef(f) === e`